src/payment/klarna-customer-experience.md

```diff
@@ -4,22 +4,22 @@ title: Klarna Storefront Experience
 
 Klarna appears as a payment method during checkout, if it is enabled in the configuration. When data sharing is used, Klarna can instantly prequalify the customer and present the available solutions.
 
-![Pay over time with Klarna]({% link images/images/storefront-cart-payment-method-klarna2.png %}){: .zoom}
-_Storefront - Pay over time with Klarna_
+![Pay later with Klarna]({% link images/images/storefront-cart-payment-method-klarna_ot.png %}){: .zoom}
```

Comment:
If you are changing images, please delete the original image (If it is not used somewhere else).

Reply:
Done, please check again

Reply:
Hi again! Is there anything left from my side? @dobooth